spec/index.bs

```diff
@@ -4896,14 +4899,15 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                         : {{GPUBufferBindingType/"read-only-storage"}}
                         :: |variable| is declared with the storage class `storage` and access mode `read`.
                     </dl>
+
                 ::
-                    If |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/minBindingSize}} is not `0`:
-                    - If the last field of the corresponding structure defined in the shader has an unbounded array type,
-                        then the value of |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/minBindingSize}}
-                        must be greater than or equal to the byte offset of that field plus the stride of the unbounded array.
-                    - If the corresponding shader structure doesn't end with an unbounded array type,
-                        then the value of |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/minBindingSize}}
-                        must be greater than or equal to the size of the structure.
+                    If |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/minBindingSize}} is not `0`,
+                    then it must be at least the [=minimum binding size=] for the associated buffer variable in the shader.
+                    If the variable has [=store type=] |T|, the minimum binding size is SizeOf(|T|).
+                    In this calculation, if |T| is a [=runtime-sized=] array or contains a runtime-sized array,
+                    that array is assumed to have one element.
+                    Enforcing this lower bound ensures any memory location accessible via the buffer variable will lie
+                    within the mapped buffer region.
```

Comment:
let's not use the wording "mapped buffer region" here to avoid confusion with the actual buffer mapping

Reply:
It's an explanatory / motivating note, so I could drop the sentence entirely.

Or, I came up with this rephrasing:

> Enforcing this lower bound ensures that, when this shader is invoked by a draw or dispatch command, any memory access via the buffer variable will only access memory locations inside the region of the buffer mapped to the binding in that draw or dispatch command.

That seems a bit awkward.  I could go either way.

Reply:
I finally understand the problem.  "Mapping" is about revealing the buffer storage via the host address space. That's a different concept entirely. 

Changed it to:

>  Enforcing this lower bound ensures reads and writes via the buffer variable only access memory locations within the bound region of the buffer.